tests/yum.txt

```diff
@@ -0,0 +1,136 @@
+#!/bin/sh
```

Comment:
Thanks a lot for writing the tests. I know that requires a quite amount of time. Much appreciated.

Reply:
It's a pleasure, thank you for pacapt :+1: